src/containers/ConversionForm/index.tsx

```diff
@@ -80,18 +91,71 @@ const ConversionForm = ({
                         setFileToConvert(file);
                     }}
                     customRequest={(options) =>
-                        customRequest(fileToConvert, receiveFileToConvert, setError, options)
+                        customRequest(
+                            fileToConvert,
+                            receiveFileToConvert,
+                            setError,
+                            options
+                        )
                     }
                 >
                     <Button type="default">Select file</Button>
                 </Upload>
             </div>
-            <Divider orientation="right" orientationMargin={400}>
-                {" "}
-            </Divider>
+            <Collapse accordion className={styles.collapse} bordered={false}>
+                <Panel
+                    key="placeholder"
+                    header=""
+                    showArrow={false}
+                    collapsible="disabled"
+                ></Panel>
+                <Panel
+                    key="placeholder"
+                    header=""
+                    showArrow={false}
+                    collapsible="disabled"
+                ></Panel>
+            </Collapse>
+            {fileToConvert ? (
+                <div>
+                    <h3 className={styles.provide}>
+                        Provide display information (optional)
+                    </h3>
+                    <h3 className={styles.convertText}>
+                        You can import your model now with defaults, or specify
```

Comment:
you shouldn't use <h3> for all different text sizes. Use the semantically appropriate tag